.code-samples.meilisearch.yaml

```
@@ -3,6 +3,42 @@
 # the documentation on build
 # You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
 ---
+get_all_tasks_filtering_1: |-
+  client.index("movies").getTasks() { result in
+    switch result {
+    case .success(let taskResult):
+      print(taskResult)
+    case .failure(let error):
+      print(error)
+    }
+  }
+get_all_tasks_filtering_2: |-
+  client.index("movies").getTasks(params: TasksQuery(status: ["succeeded", "failed"], types: ["documentAdditionOrUpdate"])) { result in
+    switch result {
+    case .success(let taskResult):
+      print(taskResult)
+    case .failure(let error):
+      print(error)
+    }
+  }
```

This one is not done on an index

```suggestion
get_all_tasks_filtering_2: |-
  client.getTasks(params: TasksQuery(status: ["succeeded", "failed"], types: ["documentAdditionOrUpdate"])) { result in
    switch result {
    case .success(let taskResult):
      print(taskResult)
    case .failure(let error):
      print(error)
    }
  }
```